st/clitests/s3iamcli_test_config.yaml

```diff
@@ -21,4 +21,4 @@
 
 # LDAP credentials used for creating s3 credentials
 ldapuser: 'sgiamadmin'
-ldappasswd: 'ldapadmin'
+ldappasswd: 'Admin@123'
```

Comment:
I guess we do not want to have this file change as part of the PR !!!

Reply:
Removed these changes, not required for PR.